core/templates/dev/head/components/question-directives/question-player/question-player.directive.ts

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Controller for the questions player directive.
  */
 
-require('domain/question/QuestionPlayerBackendApiService.ts');
+require('domain/question/QuestionBackendApiService.ts');
 require('domain/utilities/UrlInterpolationService.ts');
```

Comment:
Please move these two in between the directive requires and constants requires (line 103).

Reply:
Done.